tools/pv_tree_generator/constants.py

```diff
@@ -42,6 +42,23 @@
     ('NAICS/99', 'Unclassified')
 ])
 
+NACE = collections.OrderedDict([
```

Comment:
Some of the title is too long to fit in the menu region. Where do you get them, can discuss if they have shorter names.

Reply:
Got it from @eftekhari-mhs. I'll check with her to see if we can make the title shorter.